taichi/runtime/llvm/llvm_context_pass.h

```diff
@@ -174,6 +174,39 @@ struct AMDGPUAddStructForFuncPass : public ModulePass {
   }
 };
 
+struct AMDGPUConvertFunctionBodyAllocsAddressSpacePass : public FunctionPass {
```

Comment:
Add some comments to explain what modifications were applied to the original `AllocaInst`

Reply:
Thanks for the discussion with @jim19930609, a better way is to use `hipfreeasync`. Due to the rocm version problem, I think we can switch to the `hipfreeasync` implementation afterwards.